src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -124,8 +124,10 @@ public Dac findById(Integer dacId) {
         Dac dac = dacDAO.findById(dacId);
         List<User> chairs = dacDAO.findMembersByDacIdAndRoleId(dacId, UserRoles.CHAIRPERSON.getRoleId());
         List<User> members = dacDAO.findMembersByDacIdAndRoleId(dacId, UserRoles.MEMBER.getRoleId());
-        dac.setChairpersons(chairs);
-        dac.setMembers(members);
+        if (dac != null) {
+            dac.setChairpersons(chairs);
+            dac.setMembers(members);
+        }
         return dac;
```

Comment:
I would only return the dac if not null (and I like to use `Objects.nonNull` for this). Something like:
```suggestion
        if (Objects.nonNull(dac)) {
            dac.setChairpersons(chairs);
            dac.setMembers(members);
            return dac;
        }
        throw new NotFoundException("Could not find DAC with the provided id: " + dacId);
```